src/components/PopupWithForm.js

```diff
@@ -30,19 +40,13 @@ class PopupWithForm extends Popup {
 
   open() {
     super.open();
-    this._button.textContent = "Save";
+    this._popupForm.reset();
   }
 
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`